script/Ciel/Day 5/05_01_CIEL05_4B.txt

```diff
@@ -209,27 +214,35 @@ However―――
 [sha:73176aa278552bbeda59821351045e7e8c94a89e]{
 -- Page 744, Offset 12877.
 -- だが、そのすべての直感を排斥して、ヤツは地を這う俺を選んだ。
-However, rejecting all of his intuition, he goes for me who is creeping on the ground.
+// Another instance of just _awkward_ state tagging in EN with 地を這う.
+// Really needs to be broken out into another thought or just dropped.
+However, in a complete rejection of his intuition, he chooses me instead.
```

Comment:
prefer introducing em dash but colon could work too
me instead--the one creeping across the ground toward him.
me: the one creeping across the ground toward him.